src/components/StaticHeader/StaticHeader.jsx

```diff
@@ -27,4 +28,12 @@ const StaticHeader = (props) => (
     </HeaderOuter>
 );
 
+StaticHeader.defaultProps = {
+    children: ""
+};
+
+StaticHeader.propTypes = {
+    children: PropTypes.string
```

Comment:
The code here is kind of strange, not sure it's written this way. In general, we'd expect `children` to be just anything renderable (i.e. `PropTypes.node`), but here I agree that a string is what we want.